src/NuGet.Clients/NuGet.Console/Xamls/ConsoleContainer.xaml.cs

```diff
@@ -75,6 +80,26 @@ public void NotifyInitializationCompleted()
 
         public void Dispose()
         {
+            if (!_isTelemetryEmitted)
+            {
+                var telemetryEvent = new TelemetryEvent("PackageManagerConsoleLoadCount", new Dictionary<string, object>
+                                {
+                                    { "NugetPMCLoadCount", _windowLoadCount}
+                                });
+                TelemetryActivity.EmitTelemetryEvent(telemetryEvent);
+
+                // Work around to detect if PMC loaded automatically because it was last focused window.
+                if (IsLoaded)
+                {
+                    telemetryEvent = new TelemetryEvent("PackageManagerConsoleDefaultOpen");
```

Comment:
Does this really mean "Default Open"? IsLoaded is just telling you that PMC has been loaded at some point, but do you really know when?

Reply:
Default open: PMC loaded automatically because it was last focused window before we close VS.
Instead of trying to figure out when did PMC was opened (which is hard), we solve problem from backward.
It  just detect if PMC is active/focused window before VS close, so it's analogues to same problem above.

Reply:
Hmm...what happens when a user installs a package with an install.ps1 and the PMC initializes, and then they open PMC and focus on it? 

Does it send this event?

Reply:
When you install package with an install.ps1 then it only initialize Powershell only, not PMC. So if you open PMC later and focus on it then  it's separate event so it'll be sent.

Reply:
Ok, so now if I understand, you're trying to write that it WILL BE opened by default when VS is opened next time? In that case, I suggest something that is less confusing without that context. Maybe "WillReopen" or "OpenAtShutdown" or something?

Reply:
Actually PMUI and PMC creates separate instances of `PowerShellHost.cs`.

Reply:
Ok. How about "ReOpenAtStart"? I believe this one better reflect what it does.

Reply:
Yeah, that's a better phrasing. Maybe "ReopenOnStartup"?